blocks/bricks/recurrent.py

```diff
@@ -962,18 +948,18 @@ def apply(self, *args, **kwargs):
         low_memory : bool
             Use the slow, but also memory efficient, implementation of
             this code.
-
-        See docstring of the class for arguments appearing in
-        self.apply.sequences, self.apply.states, self.apply.contexts
-        All arguments values are of type :class:`~tensor.TensorVariable`.
-
-        In addition the `iterate`, `reverse`, `return_initial_states` or
-        any other argument defined in `recurrent_apply` wrapper.
+        sequences, states, contexts : :class:`~tensor.TensorVariable`
```

Comment:
The `apply` method does not have arguments `sequences`, `states`, `contexts`. It has `*args` and `**kwargs` and you can explain how those are interpreted.